openstudiocore/src/utilities/core/ApplicationPathHelpers.cxx.in

```diff
@@ -141,36 +153,32 @@ namespace openstudio {
 
   openstudio::path getOpenStudioModule()
   {
-    openstudio::path openstudioDirPath;
+    openstudio::path path;
     #if defined _WIN32
       TCHAR szPath[MAX_PATH];
       if( GetModuleFileName( GetCurrentModule(), szPath, MAX_PATH ) ) {
         LOG_FREE(Debug, "ApplicationPathHelpers", "getOpenStudioModule, szPath = '" << szPath << "'");
-        openstudioDirPath = completeAndNormalize(toPath(szPath));
+        path = toPath(szPath);
       }
     #else
       Dl_info info;
       if (dladdr("GetCurrentModule", &info)) {
         LOG_FREE(Debug, "ApplicationPathHelpers", "getOpenStudioModule, info.dli_fname = '" << info.dli_fname << "'");
-        auto path = toPath(info.dli_fname);
-        if ( path.parent_path().empty() ) {
-          std::istringstream pathstream( getenv("PATH") );
-          std::string pathstring;
-          while ( std::getline(pathstream, pathstring, ':') ) {
```

Comment:
@kbenne did you add this?  Why were we having to fall back to searching in PATH?  That doesn't seem right to me

Reply:
Evan did: https://github.com/jmarrec/OpenStudio/commit/93572bb0ce9c2a454413c5d84a558d3c7fb0ea7f

Reply:
We should add your comment from above to this file to remind us:

> let's call assume the getApplicationPath (which uses GetModuleFileName (windows) or dladdr (Unix))returns just OpenStudioApplicationSymlink.
> 
> You first try to locate that OpenStudioApplicationSymlink in the current directory, then if you cannot, it means it's in the system PATH. There's no way around it IMHO. This wouldn't be a concern unless you are launching a symlink that lives in your PATH, and outside the directory where the symlink is Cf my other comment on the PR (outside of review box)
> 
> ./OpenStudioApplicationSymlink will work
> /home/OpenStudioApplicationSymlink will work
> OpenStudioApplicationSymlink itself will not work if you aren't in the same folder

Reply:
Done